### PR TITLE
*: lokoctl {cluster,component} install -> lokoctl {cluster,component} apply 

### LINKED ISSUE #128: Rename install to something idempotent-like
We currently have the command `lokoctl cluster install` to install the cluster and to apply changes to it.

We should rename `install` to something else like `apply` or `converge` which conveys the meaning of what we're doing better.

### PR DESCRIPTION
**Pull request by iaguis**
This aligns more with the idempotency of this operation.

Check commit messages for details.

Closes #128

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /docs @johananl @surajssd
 *.md @johananl @surajssd
 /ci @pothos @invidian
-/pkg/install @invidian @johananl @mauriciovasquezbernal @ipochi
+/pkg/apply @invidian @johananl @mauriciovasquezbernal @ipochi
```
Comment (invidian) [inline CODEOWNERS]: That doesn't seem like a good package name at all to me :smile: We should move function from this package to the better places and remove it instead I think. Can we not rename it until refactoring happens?

File: cli/cmd/component.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ import (
 
 var componentCmd = &cobra.Command{
 	Use:   "component",
-	Short: "Install Lokomotive components",
+	Short: "Operate Lokomotive components",
```
Comment (invidian) [inline cli/cmd/component.go]:
```suggestion
	Short: "Manage Lokomotive components",
```

File: docs/cli/lokoctl.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Manage Lokomotive clusters.
 ### SEE ALSO
 
 * [lokoctl cluster](lokoctl_cluster.md)	 - Manage a Lokomotive cluster
-* [lokoctl component](lokoctl_component.md)	 - Install Lokomotive components
+* [lokoctl component](lokoctl_component.md)	 - Operate Lokomotive components
```
Comment (invidian) [inline docs/cli/lokoctl.md]:
```suggestion
* [lokoctl component](lokoctl_component.md)	 - Manage Lokomotive components
```

File: docs/cli/lokoctl_cluster.md [hunk 4]
```diff
@@ -23,6 +23,6 @@ Manage a Lokomotive cluster
 ### SEE ALSO
 
 * [lokoctl](lokoctl.md)	 - Manage Lokomotive clusters.
+* [lokoctl cluster apply](lokoctl_cluster_apply.md)	 - Apply configuration changes to a Lokomotive cluster with components
```
Comment (invidian) [inline docs/cli/lokoctl_cluster.md]: Indentation?

File: cli/cmd/cluster-apply.go [hunk 5]
```diff
@@ -31,6 +31,7 @@ var (
 	skipComponents bool
 )
 
+//nolint:gochecknoglobals
```
Comment (invidian) [inline cli/cmd/cluster-apply.go]: Won't be needed because of #141

File: cli/cmd/component-apply.go [hunk 6]
```diff
@@ -25,10 +25,13 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/config"
 )
 
+//nolint:gochecknoglobals
```
Comment (invidian) [inline cli/cmd/component-apply.go]: Ditto #141

File: CODEOWNERS (hunk 1 above)
Comment (iaguis) [reply CODEOWNERS]: Yeah, you're right. I'll keep as install for now then.

File: docs/configuration-reference/components/rook-ceph.md [hunk 7]
```diff
@@ -11,7 +11,7 @@
 
 ## Introduction
 
-[Rook](https://rook.io/) is a storage orchestrator for Kubernetes. This component installs a Ceph
+[Rook](https://rook.io/) is a storage orchestrator for Kubernetes. This component applys a Ceph
```
Comment (invidian) [inline docs/configuration-reference/components/rook-ceph.md]:
```suggestion
[Rook](https://rook.io/) is a storage orchestrator for Kubernetes. This component applies a Ceph
```

Comment (invidian) [reply docs/configuration-reference/components/rook-ceph.md]: This is actually not correct. I think `installs` is correct in this case.

File: docs/configuration-reference/components/rook.md [hunk 8]
```diff
@@ -15,7 +15,7 @@
 Kubernetes, providing the platform, framework, and support for a diverse set of storage solutions to
 natively integrate with cloud-native environments.
 
-This component installs the Rook operator.
+This component applys the Rook operator.
```
Comment (invidian) [inline docs/configuration-reference/components/rook.md]: Ditto.

File: docs/configuration-reference/components/rook-ceph.md (hunk 7 above)
Comment (iaguis) [reply docs/configuration-reference/components/rook-ceph.md]: heh, search and replace :see_no_evil:

**Pull request merged**